datadog_checks_base/tests/compose/kerberos-agent.yaml

```diff
@@ -0,0 +1,90 @@
+version: '3.7'
+
+services:
+
+  agent:
+    image: agent
+    build: ./kerberos-agent
+    environment:
+      DD_API_KEY: $DD_API_KEY
+      KRB5_KEYTAB: ${KRB5_KEYTAB}
+      KRB5_CCNAME: ${KRB5_CCNAME}
+      KRB5_REALM: ${KRB5_REALM}
+      KRB5_SVC: ${KRB5_SVC}
+      KRB5_KDC: kerberos-kdc
+      WEBHOST: ${WEBHOST}
+      DOMAIN: example.com
+      KRB5_USER: datadog/admin
+      KRB5_PASS: 123pass
```

Comment:
(nit) Should we have it as environment variables as well?

Reply:
It's not very secure, but we use it in our `setupkeytab.sh` script like so:

`echo ${KRB5_PASS} | kinit user/nokeytab@${KRB5_REALM} -c ${KRB5_CCNAME}/tkt_nokeytab -V`